Sources/NostrSDK/EventCreating.swift

```diff
@@ -340,11 +340,12 @@ public extension EventCreating {
     /// Creates a ``DateBasedCalendarEvent`` (kind 31922) which starts on a date and ends before a different date in the future.
     /// Its use is appropriate for all-day or multi-day events where time and time zone hold no significance. e.g., anniversary, public holidays, vacation days.
     /// - Parameters:
-    ///   - name: The name of the calendar event.
+    ///   - identifier: A unique identifier for the calendar event. Can be reused in the future for replacing the calendar event. If an identifier is not provided, a ``UUID`` string is used.
+    ///   - title: The title of the calendar event.
     ///   - description: A detailed description of the calendar event.
     ///   - startDate: An inclusive start date. Must be less than end, if it exists. If there are any components other than year, month,
     ///   - endDate: An exclusive end date. If omitted, the calendar event ends on the same date as start.
-    ///   - location: The location of the calendar event. e.g. address, GPS coordinates, meeting room name, link to video call.
+    ///   - locations: The locations of the calendar event. e.g. address, GPS coordinates, meeting room name, link to video call.
```

Comment:
My NIP-52 amendment in the NIPs repo changes calendar events to allow multiple locations.